packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -335,6 +338,36 @@ export function getValidatorApi({
       );
   }
 
+  async function updateDefaultGraffiti(): Promise<void> {
+    defaultGraffiti = await getDefaultGraffiti();
+  }
+
+  async function getDefaultGraffiti(): Promise<string> {
+    const consensusCode = ClientCode.LS;
+    const consensusCommit = opts.version ?? "";
+
+    const lodestarClientVersion = {
+      code: consensusCode,
+      name: "Lodestar",
+      version: opts.version ?? "",
+      commit: consensusCommit,
+    };
+
+    const executionClientVersions = await chain.executionEngine
+      .getClientVersion(lodestarClientVersion)
+      .catch((_) => []);
+
+    if (executionClientVersions.length !== 0) {
+      const executionCode = executionClientVersions[0].code;
+      const executionCommit = executionClientVersions[0].commit;
+
+      return `${executionCode}|${executionCommit.slice(0, 2)}|${consensusCode}|${consensusCommit.slice(0, 2)}`;
+    }
+
+    // No EL client info available. We still want to include CL info albeit not spec compliant
+    return `${consensusCode}|${consensusCommit.slice(0, 2)}`;
```

Comment:
Alternative to this would be to return our current default graffiti which is what Lighthouse does but this seems good to me as well